pom.xml

```diff
@@ -335,6 +335,16 @@
         <enabled>true</enabled>
       </snapshots>
     </pluginRepository>
+    <!-- configure the EBR plug-in repo -->
+    <pluginRepository>
+      <id>ebr</id>
+      <url>https://repo.eclipse.org/content/repositories/ebr/</url>
```

Comment:
i.e. EBR for me is an osgi repository format similar to p2 thus i'm pretty sure i'm missing some context. i.e. docs or links to what this is.

Googling I assume you mean *eclipse* EBR (https://www.eclipse.org/ebr/) which has like *zero* information.

Could you provide some background links/info ?